test/Microsoft.DotNet.Installer.Tests/installer-tests-prereqs.proj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.Build.Traversal">
+
+  <PropertyGroup>
+    <TargetFramework>$(NetCurrent)</TargetFramework>
+  </PropertyGroup>
+
+  <!-- Make sure that Microsoft.DotNet.UnifiedBuild.Tasks is built before building the scenario-tests project -->
+  <ItemGroup>
+    <ProjectReference Include="$(TasksDir)Microsoft.DotNet.UnifiedBuild.Tasks/Microsoft.DotNet.UnifiedBuild.Tasks.csproj" BuildInParallel="false" />
+    <ProjectReference Include="$(RepoProjectsDir)scenario-tests.proj" BuildInParallel="false" />
```

Comment:
```suggestion
    <ProjectReference Include="$(RepoProjectsDir)scenario-tests.proj" BuildInParallel="false" AdditionalProperties="SkipScenarioTestsDependencies=true" />
```

So that this works outside of YML as well. You can remove the property from YML with this.

Reply:
Nice, thanks!